### PR TITLE
Add unsound `prettytable-rs`

### PR DESCRIPTION
**Pull request by david0u0**

### CODE REVIEW

File: crates/prettytable-rs/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+```toml
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "prettytable-rs"
+date = "2022-12-27"
```
Comment (pinkforest) [inline crates/prettytable-rs/RUSTSEC-0000-0000.md:5]: The date is set when it was originally reported where we align with other databases doing this backdating.

**Comment by pinkforest** [comment]
Thanks, will merge today/tomorrow after 1.0..0 is out.

File: crates/prettytable-rs/RUSTSEC-0000-0000.md (hunk 1 above)
Comment (david0u0) [reply crates/prettytable-rs/RUSTSEC-0000-0000.md]: fixed

**Comment by pinkforest** [comment]
fyi - ended up releasing 0.10.0 - fixed tihs one to refer to 0.10.0

**Pull request merged**